code/__HELPERS/logging/admin.dm

```diff
@@ -36,8 +39,7 @@
  * Mirrors this log entry to log_access when access_log_mirror is TRUE, so this proc
  * doesn't need to be used alongside log_access and can replace it where appropriate.
  */
-/proc/log_suspicious_login(text, access_log_mirror = TRUE)
-	if (CONFIG_GET(flag/log_suspicious_login))
-		WRITE_LOG(GLOB.world_suspicious_login_log, "SUSPICIOUS_ACCESS: [text]")
+/proc/log_suspicious_login(text, list/data, access_log_mirror = TRUE)
+	logger.Log(LOG_CATEGORY_SUSPICIOUS_LOGIN, text)
```

Comment:
This doesn't respect the config flag anymore, which makes either the proc's documentation inaccurate, or is a regression.